### DISCUSSION TITLE
Change of Ingress class on a running kafka cluster.

### DISCUSSION THREAD

**Discussion by Sai-Charan-Madhvaraj** [strimzi/strimzi-kafka-operator#10154] [Q&A]
Hi Team, 

Is it advisable to change the ingress class (kafka.spec.kafka.listeners.configuration.class) on a running kafka cluster to another ingress class ? 

FYI - Tried this in local setup and seems to work fine with out any issues, 
But would like to know 
1.  any other complication might arise by changing ?
2. Any other additional configuration  changes required ?

**Comment by scholzj** [comment]
I never tried it, but I don't see any issues with that.
